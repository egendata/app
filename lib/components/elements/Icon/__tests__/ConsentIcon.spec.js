import React from 'react'
import { shallow } from 'react-native-testing-library'
import MockedProvider from '../../../../utils/test-utils'
import ConsentIcon from '../../Icon/Icon'

describe('elements/Icons/QRIcon', () => {
  it('should render', () => {
    const { output } = shallow(
      <MockedProvider>
        <ConsentIcon />
      </MockedProvider>,
    )

    expect(output).toMatchSnapshot()
  })
})
