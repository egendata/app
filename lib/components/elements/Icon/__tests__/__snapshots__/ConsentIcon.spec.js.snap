// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`elements/Icons/QRIcon should render 1`] = `
<ThemeProvider
  theme={
    Object {
      "colors": Object {
        "accept": "#0AC238",
        "border": "#EBEAFA",
        "deny": "#FF5374",
        "inactive": "#E0E7EB",
        "lightGrey": "#EAEAF1",
        "loading": "#81A1B1",
        "primary": "#0099E5",
        "quiet": "#FAFAFA",
        "shadow": "grey",
        "text": Object {
          "pre": "#81A1B1",
          "primary": "#454F54",
        },
        "white": "#FFFFFF",
      },
      "fontSize": Object {
        "large": "48px",
        "medium": "32px",
        "regular": "18px",
        "regularMedium": "24px",
        "small": "14px",
      },
      "sizes": Object {
        "elevation": 6,
      },
      "spacing": Object {
        "large": "48px",
        "medium": "24px",
        "small": "12px",
      },
      "typography": Object {
        "bold": "Karla-Bold",
        "contrast": "Rubik-Medium",
        "regular": "Karla-Regular",
      },
    }
  }
>
  <UNDEFINED />
</ThemeProvider>
`;
